apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js

```diff
@@ -147,6 +148,10 @@ var MozMobileMessageShim = {
     return mozMobileMessage.retrieveMMS(id).then((message) => {
       // Return void instead of message to avoid clone error issue.
       return;
+    }, (error) => {
+      // Clone the error if error is DOMError. Should be removed once
+      // https://github.com/gaia-components/bridge/issues/80 addressed.
```

Comment:
actually I think it's easier of we request DOMError (and maybe DOMException) to be cloneable; can you file a bug on Gecko (in Core::DOM) and add the bug number here (but keep the link to the bridge issue) ?